.github/workflows/deploy-grafana-dashboards.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy grafana dashboards
+
+on: workflow_dispatch
+
+jobs:
+  deploy_grafana:
+    runs-on: ubuntu-20.04
+    strategy:
+      # Don't stop other deployments if one fails
+      fail-fast: false
+      matrix:
+        include:
+          # To enable auto-deployments of grafana dashboards
+          # for other clusters add its name and, a GitHub
+          # secret for it and save its token to `grafana_token`
+          - cluster_grafana_name: grafana.pilot
+            grafana_token: secrets.GRAFANA_TOKEN
```

Comment:
I do not think we are using GitHub secrets on other workflows... maybe we want to keep that state and have all these secrets in some of the existing (or maybe a new) encrypted file?

Reply:
@sgibson91's work in https://github.com/2i2c-org/pilot-hubs/pull/726 means we already have a space for per-cluster sops encrypted secrets (secrets/config). Let's use that? But that means we'll need a python script or something that can decrypt the secrets and use them. But that's cleaner I think.